### PR TITLE
Move select & delete icons to right side

### PR DESCRIPTION
**Pull request by debbieellis7**
## Description

Issue: #1200 

Check accessibility and move font manager 'select & delete' icons to the right side.

## Checklist:
- [x] I've tested the code.
- [ ] My code is easy to read, follow, and understand
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation / docblocks.

### CODE REVIEW

File: src/assets/js/react/components/FontManager/AddUpdateFontFooter.js [hunk 1]
```diff
@@ -160,43 +160,50 @@ export class AddUpdateFontFooter extends Component {
...
-            className='button gfpdf-button primary cancel'
-            onClick={onHandleCancelEditFont}
-            onKeyDown={onHandleCancelEditFontKeypress}
-            tabIndex={tabIndex}
-          >
-            {GFPDF.fontManagerCancelButtonText}
+        <div className='buttons-icons'>
+          <div>
+            {id && (
+              <div
```
Comment (jestonihpi) [inline src/assets/js/react/components/FontManager/AddUpdateFontFooter.js:166]: Add a button role for this divider. e.g role="button"

File: src/assets/js/react/components/FontManager/AddUpdateFontFooter.js [hunk 2]
```diff
@@ -160,43 +160,50 @@ export class AddUpdateFontFooter extends Component {
...
-            onClick={() => this.handleDeleteFont(id)}
-            onKeyDown={e => this.handleDeleteFontKeypress(e, id)}
-            tabIndex={tabIndex}
-          />
-        )}
 
-        {loading && <Spinner style='add-update-font' />}
+          <div className='select-delete-icons'>
+            {id && (
+              <span
```
Comment (jestonihpi) [inline src/assets/js/react/components/FontManager/AddUpdateFontFooter.js:189]: Add button role on this one too.

File: src/assets/js/react/components/FontManager/AddUpdateFontFooter.js [hunk 3]
```diff
@@ -160,43 +160,50 @@ export class AddUpdateFontFooter extends Component {
...
+                className={'dashicons dashicons-yes' + selectedBoxStyle}
+                onClick={() => this.handleSelectFont(id, selectedFont)}
+                onKeyDown={e => this.handleSelectFontKeypress(e, id, selectedFont)}
+                tabIndex={tabIndex}
+              />
+            )}
+
+            {id && (
+              <span
```
Comment (jestonihpi) [inline src/assets/js/react/components/FontManager/AddUpdateFontFooter.js:198]: Missing role button on this one as well.

File: src/assets/js/react/components/FontManager/AddUpdateFontFooter.js (hunk 1 above)
Comment (jestonihpi) [reply src/assets/js/react/components/FontManager/AddUpdateFontFooter.js:166]: Yes , that would be much better, with the exception of icon buttons.

**Comment by jakejackson1** [comment]
@debbieellis7 FYI about the translations, we already have a Cancel string available https://github.com/GravityPDF/gravity-pdf/blob/3338e6bb6835c25724bb15594b4e9c261179e1fb/src/Helper/Helper_Data.php#L231 so no need to add a new one. We should also still with the existing naming conventions https://github.com/GravityPDF/gravity-pdf/blob/3338e6bb6835c25724bb15594b4e9c261179e1fb/src/Helper/Helper_Data.php#L300-L305

I'd suggest `fontManagerUpdateAriaLabel` etc

**Pull request merged**